profiles/common/themes/ec_resp/css/interinstitutional_less/hack-search.less

```diff
@@ -46,5 +60,20 @@
     height:16px;
     font-size:100%;
     overflow:hidden;
-  }
+  } 
+  input::-webkit-input-placeholder { /* WebKit, Blink, Edge */
+    color:    #555;
+}
+input:-moz-placeholder { /* Mozilla Firefox 4 to 18 */
+   color:    #555;
+   opacity:  1;
+}
+input::-moz-placeholder { /* Mozilla Firefox 19+ */
+   color:    #555;
+   opacity:  1;
+}
+input:-ms-input-placeholder { /* Internet Explorer 10-11 */
+   color:    #555;
+}
```

Comment:
This section has some coding standards problems:
* Documentation should be above the ruleset, not inline.
* Indentation is not correct.
* There should only be a single space following the colon of a property.
* No blank lines inside rulesets.